modules/tinymce/src/core/main/ts/textpatterns/core/Pattern.ts

```diff
@@ -115,10 +116,23 @@ const fromRawPatterns = (patterns: RawPattern[]): Pattern[] => {
   return normalized.values;
 };
 
+const fromRawPatternsLookup = (lookupFn: (ctx: DynamicPatternContext) => RawPattern[]): DynamicPatternsLookup => {
+  return (ctx: DynamicPatternContext) => {
+    const rawPatterns = lookupFn({
+      text: ctx.text,
+      allowTrailingSpaces: ctx.allowTrailingSpaces,
+      block: ctx.block,
+    });
```

Comment:
```suggestion
    const rawPatterns = lookupFn(ctx);
```
I believe this can just be simplified to this? I think the original intention was to make sure we didn't expose something public we didn't want to. If that was the case though then we should have a different type in `lookupFn`.